### ISSUE TITLE
Add more tools/options

### ISSUE THREAD

**Issue by sergeylappo** [sergeylappo/boox-rapid-draw#16]
Allow user to configure tool currently used for drawing

**Comment by jdkruzr** [comment]
does this mean being able to change the thickness/color of the overlay ink?

**Comment by iWhatty** [comment]
Yes, I've added this as a new feature on my local branch. If this project is still alive, I can submit a pull request.
Right now, I'm testing a new menu with a few basic colors and stroke sizes.

**Comment by jdkruzr** [comment]
oh absolutely, man. I've been promoting this app to everyone I know who owns a Boox, lol -- especially since Onyx changed what was in /onyxconfig/eac_config to /onyxconfig/mmkv/onyx_config and made it partially binary and useless for doing ink optimization.

**Comment by sergeylappo** [comment]
Hey everyone!
Project is alive, please submit a PR.

Autumn/winter in NL is depressing, so I have limited energy for side activities + also quite busy time at work.

TL;DR any contributions are welcome ;)
